rules/do-you-set-a-specific-time-to-follow-up-a-prospect/rule.md

```diff
@@ -1,35 +1,42 @@
 ---
 type: rule
-archivedreason: 
 title: Do you set a specific time to follow up a prospect?
-guid: 696bd745-346c-419b-b740-a58569d94a69
 uri: do-you-set-a-specific-time-to-follow-up-a-prospect
-created: 2012-08-30T13:06:15.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
-- title: Ulysses Maclaren
-  url: https://ssw.com.au/people/ulysses-maclaren
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Ulysses Maclaren
+    url: https://ssw.com.au/people/ulysses-maclaren
 related: []
 redirects: []
-
+created: 2012-08-30T13:06:15.000Z
+archivedreason: null
+guid: 696bd745-346c-419b-b740-a58569d94a69
 ---
-
 Sometimes the prospect is not ready at the end of the initial meeting to engage you for a spec review or ad-hoc work. Even if this is the case, it's really important you leave the meeting with a crystal clear future about the next stage and when that will be. Every prospect needs an intelligently scheduled follow-up activity.
 
 <!--endintro-->
 
 So, at the end, ask the prospect:
-                                                            "When would be a good time to meet next?"
-                       Next Friday
-                      "Cool, speak then"
-                                                                   Figure: Bad example: The client really won't remember this                                                                                                     "When would be a good time to meet next?"
-                       Next Friday
-                       "Do you have your calendar?"
-                        Sure, hold on
-                       "What time suits best?"
-                       Let's say 3pm
-                       "Cool, Do you want to send the appointment or should I?"                                                                  Figure: Good example: Asking them lots of questions and setting a specific time and date for the next meeting (or phone call)                                                          
+
+"When would be a good time to meet next?" Next Friday
+
+"Cool, speak then".
+
+**Figure: Bad example: The client really won't remember this**        
```

Comment:
```suggestion
::: greybox  
- "When would be a good time to meet next?" Next Friday
- "Cool, speak then".
:::
**Figure: Bad example: The client really won't remember this**        
```

something like this might look a little nicer? what do you think?